src/workflow/ArcanistSubmitWorkflow.php

```diff
@@ -18,6 +18,9 @@
   private $submitQueueUri;
   private $submitQueueClient;
 
+  const REFTYPE_BRANCH = 'branch';
```

Comment:
This file must have gotten rebased into my commit.  I'm  going to try this PR and resubmit a clean one.

About this change, looks like some cut and paste from ArcanistLandWorkflow.php is what happened and the author forgot the constants so this never worked before as far as I can tell.  Maybe we need a php linter :)

In ArcanistLandWorkflow.php there are actually 2 different switch statements that use it so having a class var for this makes sense there.  I am okay with sticking with the constant so the code looks consistent.